.github/workflows/release-embedding-sdk.yml

```diff
@@ -4,32 +4,87 @@ on:
   workflow_dispatch:
     inputs:
       # TODO: Add a version validation, so the workflow won't fail when publishing to npm
-      sdk_version:
-        description: "SDK version (e.g. 0.1.3)"
-        type: string
-        required: true
-      git_ref:
-        description: "A git commit hash that will be used to build the SDK"
-        type: string
+      branch:
+        description: "Branch we want to release the SDK from"
+        type: choice
         required: true
+        default: "master"
+        options:
+          - master
+          - release-x.51.x
 
 concurrency:
-  # We want to ensure only one job is running at a time because
+  # We want to ensure only one job is running at a time per branch because
   # there could be a conflict when updating the readme file.
-  group: ${{ github.workflow }}
+  group: ${{ github.workflow }}-${{ inputs.branch }}
   cancel-in-progress: true
 
 jobs:
+  workflow-summary:
```

Comment:
This job logs this to the summary
![image](https://github.com/user-attachments/assets/b18e1e11-efdb-4991-80b7-a467a5730427)